src/Http/Controllers/OpenApiJsonController.php

```diff
@@ -35,6 +35,10 @@ protected function getJson() : array
 
     protected function configureServer(array $json) : array
     {
+        if(isset($json['servers'])) {
+            return $json;
+        }
+
```

Comment:
```suggestion
        if ((bool) config('swagger-ui.modify_file')) {
            return $json;
        }
```
can you add this instead? By adding the a `modify_file` boolean config variable, we still have the option to provide an external file where we do want to change the servers and the securitySchemes

This can be applied to `configureOAuth` method as well.

Can you also add a test for this?

Reply:
I'm a beginner in testing, can you check it out?